src/Geno/C++/GUI/Widgets/MainMenuBar.cpp

```diff
@@ -25,9 +25,11 @@
 #include "GUI/Widgets/OutputWindow.h"
 #include "GUI/Widgets/TextEdit.h"
 #include "GUI/Widgets/WorkspaceOutliner.h"
+#include "GUI/Widgets/StatusBar.h"
 #include "GUI/Modals/DiscordRPCSettingsModal.h"
 #include "Discord/DiscordRPC.h"
 
+
```

Comment:
You should look at your diffs before you stage your changes. This can easily be caught by yourself.